### PR TITLE
[Bug] [doc] The configuration of truncate is not supported when save mode is update

### PR DESCRIPTION
**Pull request by zhaomin1423**
…written on the doc #1411

## Purpose of this pull request
The configuration of truncate is not supported, but it's written on the doc

## Check list

* [x] Code changed are covered with tests, or it does not need tests for reason:
* [x] If any new Jar binary package adding in you PR, please add License Notice according
  [New License Guide](https://github.com/apache/incubator-seatunnel/blob/dev/docs/en/developement/NewLicenseGuide.md)
* [x] If necessary, please update the documentation to describe the new feature. https://github.com/apache/incubator-seatunnel/tree/dev/docs

### CODE REVIEW

File: docs/en/spark/configuration/sink-plugins/Jdbc.md [hunk 1]
```diff
@@ -61,12 +61,15 @@ Configure when `saveMode` is specified as `update` , and when the specified key
 
 Configure when `saveMode` is specified as `update` , whether to show sql
 
+### truncate
+
+Configure when `saveMode` is `overwrite`, whether to truncate table when truncate is true, otherwise to drop table.
+
```
Comment (wuchunfu) [inline docs/en/spark/configuration/sink-plugins/Jdbc.md]: @zhaomin1423 I think don't need this description.

Comment (zhaomin1423) [reply docs/en/spark/configuration/sink-plugins/Jdbc.md]: Thanks, I deleted it.

**Review by wuchunfu — APPROVED** [review]
LGTM

**Comment by wuchunfu** [comment]
Well done, looking forward to your next contribution.

**Pull request merged**